topic/timeseries/dask-weather-data-import.ipynb

```diff
@@ -57,7 +66,7 @@
    },
    "outputs": [],
    "source": [
-    "#!pip install dask pandas==2.0.0 'sqlalchemy[crate]'"
+    "!pip install dask 'pandas==2.0.0' 'crate[sqlalchemy]' 'cratedb-toolkit==0.0.10' 'pueblo>=0.0.7' kaggle"
```

Comment:
To stay consistent when it comes to library versions maybe it makes sense to always install those specified in `requirements.txt`.

Reply:
Why `pandas==2.0.0` and not a higher `pandas==2.*`?

Reply:
Using `requirements.txt` will be much better, because Dependabot does not bump inside Notebooks.

Reply:
I see. As opposed to the other notebooks here, which need `pandas<2` and `sqlalchemy<2`, this one needs more recent versions? Is that correct?